app/components/support_interface/tile_component.html.erb

```diff
@@ -1,4 +1,6 @@
-<div class="app-card<%= colour == :default ? '' : " app-card--#{colour}" %>">
-  <span class="<%= count_class %>"><%= number_with_delimiter(count) %></span>
-  <%= label %>
+<div class="<%= card_classes %>">
+  <%= href ? "<a class=\"app-card__link\" href=\"#{href}\">".html_safe : '' %>
+    <span class="<%= count_class %>"><%= number_with_delimiter(count) %></span>
+    <%= label %>
+  <%= href ? "</a>".html_safe : '' %>
```

Comment:
Either something like:
```
<% if href.present? %>
    <a class="app-card__link" href="<%= href %>">
      <span class="<%= count_class %>"><%= number_with_delimiter(count) %></span>
      <%= label %>
    </a>
  <% else %>
    <span class="<%= count_class %>"><%= number_with_delimiter(count) %></span>
    <%= label %>
  <% end %>
```
which is explicit and readable.

Or you could go with a method like:
```
def wrap_in_link_if_href_present(&block)
      if href.blank?
        concat capture(&block)
      else
        concat content_tag(:a, capture(&block), class: 'app-card__link', href: href)
      end
    end
```
and then
```
 <% wrap_in_link_if_href_present do %>
    <span class="<%= count_class %>"><%= number_with_delimiter(count) %></span>
    <%= label %>
  <% end %>
```
(very much open to better method name suggestions)

Reply:
Thanks @benswannack… your first solution I think is the most straight–forward. Updated!